lametro/templates/lametro/person.html

```diff
@@ -191,7 +191,7 @@ <h3>Committees</h3>
 
 {% block extra_js %}
 
-    {% if map_geojson %}
+    {% if map_geojson_districts %}
```

Comment:
Why check for `MAP_CONFIG` everywhere but here?

Reply:
because it looks like this section should actually only be loaded if the the districts exist.

to be honest, this code looks like it's has bug, as it would make more sense to load the relevant map for all the types of of board members, not just the commissioners. however, this current behavior